### PR TITLE
Done with project 09

### PR DESCRIPTION
**Pull request by JCardenasRdz**
Please see Human_Body_Temp_final.py for details.
This script is loaded into the jupyter notebook

### CODE REVIEW

File: Projects/09_EDA_Human_Body_Temperature/Human_Body_Temp_final.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    if pval > 0.05:
+        print('The '+ test + ' indicates that the data is NOT normally distirbuted')
+    else:
+        print('The '+ test + ' indicates that the data IS normally distirbuted')
+
+def norm_tests(xdata):
+    _, p =kstest(xdata, 'norm')
+    explain_test(p,'Kolmogorov-Smirnov test')
+    print(40*'==')
+    _, p =normaltest(xdata)
```
Comment (amirziai) [inline Projects/09_EDA_Human_Body_Temperature/Human_Body_Temp_final.py:16]: Spaces...

File: Projects/09_EDA_Human_Body_Temperature/Human_Body_Temp_final.py [hunk 2]
```diff
@@ -0,0 +1,59 @@
+import pandas as pd
+import numpy as np
+from scipy.stats import kstest, normaltest, ttest_1samp, wilcoxon, ranksums, ttest_ind
+import matplotlib.pyplot as plt
+
+def explain_test(pval,test):
```
Comment (amirziai) [inline Projects/09_EDA_Human_Body_Temperature/Human_Body_Temp_final.py:6]: Great job defining reusable functions.

**Review by amirziai — APPROVED** [review]
Great job.

http://stats.stackexchange.com/questions/103345/my-distribution-is-normal-kolmogorov-smirnov-test-doesnt-agree

File: Projects/09_EDA_Human_Body_Temperature/Human_Body_Temp_final.py (hunk 1 above)
Comment (JCardenasRdz) [reply Projects/09_EDA_Human_Body_Temperature/Human_Body_Temp_final.py:16]: Leave spaces after commas or indentations?

Comment (amirziai) [reply Projects/09_EDA_Human_Body_Temperature/Human_Body_Temp_final.py:16]: `print(40*'==')` should be `print(40 * '==')`
`_, p =normaltest` should be `_, p = normaltest`

**Pull request merged**